packages/video_player/video_player_avfoundation/example/ios/RunnerUITests/VideoPlayerUITests.m

```diff
@@ -30,15 +30,15 @@ - (void)testPlayVideo {
   XCTAssertTrue([playButton waitForExistenceWithTimeout:30.0]);
   [playButton tap];
 
-  XCUIElement *playbackSpeed1x = app.staticTexts[@"Playback speed\n1.0x"];
+  XCUIElement *playbackSpeed1x = app.staticTexts[@"1.0x\nPlayback speed"];
```

Comment:
You'll need to have both the old and new version, and wait for either one to show up, because the tests run on both stable and master.

Reply:
Yep, will do that

Reply:
Alternatively you could just look for `1.0x` which is the spirit of this check.
```suggestion
  XCUIElement *playbackSpeed1x = app.staticTexts[@"1.0x"];
```

Reply:
I missed this comment earlier. I like this better. Updated the PR with only looking for 1.0x and 5.0x